src/app/contexts/RequestContext/getOriginContext.js

```diff
@@ -1,6 +1,6 @@
 const getOriginContext = bbcOrigin => {
-  let origin = 'https://www.bbc.co.uk';
-  let isUK = true;
+  let origin = 'https://www.bbc.com';
```

Comment:
Setting .com as the default has huge affects in the application, we default to .co.uk to ensure that if the application is missing something, we dont accidentally show a UK user something like adverts. Chatting to @sareh this doesn't feel like something we should do to fix a metadata issue. Can you catch up with her please?